### ISSUE TITLE
Kuroba doesn't show up in the search results because it is a fork

### ISSUE THREAD

**Issue by pExeen** [Adamantcheese/Kuroba#366]

**Comment by K1rakishou** [comment]
Well, yes, that's how Github work. It's doesn't support search inside a forked repo either which is pretty bad.

**Comment by Adamantcheese** [comment]
Yeah, it's just how Github operates. Forks don't show up in searches, commits in a fork don't show up on your personal timeline and you can't search inside it. I've been considering moving it off of being a fork for a while now and will still keep it under consideration.
